api/tests/Api/Invitations/RejectInvitationTest.php

```diff
@@ -33,12 +33,13 @@ public function testRejectInvitationSucceedsWhenNotLoggedIn() {
                 'headers' => ['Content-Type' => 'application/merge-patch+json'],
             ]
         );
-        $this->assertResponseStatusCodeSame(200);
+        $this->assertResponseStatusCodeSame(200); // // TO DISCUSS: Wouldn't it be better to get a 204 here? The invitation doesn't really exist anymore after successful acceptance
```

Comment:
Yes, as mentioned above, I would say even a DELETE request would be appropriate for this operation

Reply:
Cleaned up in https://github.com/ecamp/ecamp3/pull/3176/commits/2a3d2ff4c1732d72fe0f52961765880892dd2da4#diff-74c41eaa1f7f42a90ba91740dcde7f1ca280e0d6960a0061070c863e0bf4a4be